.github/workflows/deploy_dbt_docs.yaml

```diff
@@ -0,0 +1,71 @@
+name: deploy-dbt-docs
+
+on:
+  workflow_run:
+    workflows: [build-and-test-dbt]
+    branches: [master, data-catalog]
+    types: [completed]
+
+jobs:
+  deploy-dbt-docs:
+    runs-on: ubuntu-latest
+    # These permissions are required to make a GitHub Pages deployment
+    permissions:
+      pages: write      # To deploy to Pages
+      id-token: write   # To verify the deployment comes from an valid source
+
+    steps:
+      # There is currently no way to early-exit a run with a success status
+      # in GitHub actions, so we have to mark each of these steps with the
+      # same `if` condition to restrict deploys to successful builds :(
+      # https://github.com/actions/runner/issues/662
```

Comment:
I _think_ just gating the job with the if statement will prevent the steps from running at all, negating the need for the early exit.

Reply:
I think that's correct as well, but I'm testing in 4a46220341271be54b98015106a13337761aba2f!

Reply:
It turns out we can't test this behavior because both [workflow_run](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run) and [workflow_dispatch](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) will only run if the workflow file exists on the default branch 🙃 

> Note: This event will only trigger a workflow run if the workflow file is on the default branch.

Are we comfortable merging this without testing and testing on prod instead? Note another downside of this limitation is we won't start building docs until we merge `data-catalog` into `master` (#55), but maybe we can bump the timeline on that issue?

Reply:
I think that's fine, nothing will break if it fails in prod except for the docs. Let's see how the checks meeting goes re: bumping #55.